cmd/admin-handlers.go

```diff
@@ -86,6 +86,61 @@ func (adminAPI adminAPIHandlers) ServiceRestartHandler(w http.ResponseWriter, r
 	sendServiceCmd(globalAdminPeers, serviceRestart)
 }
 
+// Type-safe service query params.
+type serviceQueryKey string
+
+const (
+	serviceAccessKey serviceQueryKey = "accessKey"
```

Comment:
We should send access and secretKey as part of the body insteadof query params for security reasons. On a HTTPS connection this becomes more secure just like how browser actually sets the credentials.